backend/dal/Migrations/20200401221110_initial.cs

```diff
@@ -4,12 +4,12 @@
 
 namespace Pims.Dal.Migrations
 {
-    public partial class Initial : SeedMigration
+    public partial class initial : SeedMigration
```

Comment:
For some reason, GitHub "hides" my comment in the file above but the same applies here (same class)...

We already spent some time on a previous PR to make sure the  `PostDeploy/Initial` folder was PascalCase (ie - Initial vs "initial") as this creates issues in Linux where our docker containers are built. 

Please change the name back to `*_Initial` as well as the `[Migration]` attribute. I believe that's what we use to get the path to the seed *.sql scripts...

But please @Fosol  comment on this as you created this.

Reply:
Good catch!  Rename from `initial` to `Initial`.

Reply:
Tech-debt: we should create a unit test that covers this scenario. I think it should be straightforward to check the name of the migration to be `"*_Initial"`